lib/install/engine/interactive/wizard.go

```diff
@@ -115,7 +115,7 @@ func (r *executor) waitForOperation() (operation *ops.SiteOperation, err error)
 			return trace.NotFound("no clusters created yet")
 		}
 		cluster := clusters[0]
-		operations, err := r.Operator.GetSiteOperations(cluster.Key())
+		operations, err := r.Operator.GetSiteOperations(cluster.Key(), ops.OperationsFilter{})
```

Comment:
Have you considered introducing a new API - this seems like a breaking change.
Or are you aiming at making this change transparent so it can equally be serviced by the older implementations during the upgrade?

Reply:
Correct, introducing a new API leaves a lot of room for miss-matched client and servers through the upgrade or other ports. It also implies this particular API endpoint should probably be deprecated. So by passing optional arguments, the server is just free to ignore those arguments, in which case it returns all results.